storage/doc.go

```diff
@@ -331,6 +331,29 @@ to add a [custom audit logging] header:
 	// Use client as usual with the context and the additional headers will be sent.
 	client.Bucket("my-bucket").Attrs(ctx)
 
+# Experimental gRPC API
+
+This package now includes support for the Cloud Storage gRPC API, which is now
+in preview. This implementation uses gRPC rather than the current JSON & XML
```

Comment:
```suggestion
This package now includes support for the Cloud Storage gRPC API, which is currently
in preview. This implementation uses gRPC rather than the current JSON & XML
```